projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/place-order/place-order.component.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  Component,
+  ChangeDetectionStrategy,
+  OnInit,
+  Output,
+  EventEmitter,
+  OnDestroy
+} from '@angular/core';
+
+import { CheckoutService, RoutingService } from '@spartacus/core';
+
+import { filter } from 'rxjs/operators';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'cx-place-order',
+  templateUrl: './place-order.component.html',
+  styleUrls: ['./place-order.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PlaceOrderComponent implements OnInit, OnDestroy {
+  tAndCToggler = false;
+  placeOrderSubscription: Subscription;
+
+  @Output() clearCheckoutData = new EventEmitter<void>();
```

Comment:
Instead of output just directly invoke service method. NgDestroy will take of the container reset.

Reply:
done